test/pkg.jl

```diff
@@ -182,4 +183,37 @@ temp_pkg_dir() do
         finalize(repo)
         Pkg.update()
     end
+
+    #test PkgDev redirects
+    begin
+        try
+            Pkg.register("IDoNotExist")
```

Comment:
these should have a different unexpected error in case this line succeeds somehow